### PR TITLE
refactor: to import custom props as js object

### PR DESCRIPTION
**Pull request by emmenko**
Follow up of #460 and #461

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,10 +91,10 @@ You can also access the JavaScript variables like this
 ```js
 import { customProperties } from '@commercetools-frontend/ui-kit';
```
Comment (emmenko) [inline README.md]: I think this export should point to the js object, no the json one. I also changed the export in `src/index`

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "i18n",
     "materials/custom-properties.css",
     "materials/custom-properties.json",
+    "materials/custom-properties.js",
```
Comment (emmenko) [inline package.json]: Not sure it's necessary since we are exporting it now also as a named export from `src/index`

Comment (tdeekens) [reply package.json]: Yeah I guess it's not really needed but we can keep it given anybody wants to copy it into their projects I suppose.

**Comment by emmenko** [comment]
I spoke to @montezume and we decided to mark this as a breaking change, because of the named export now returning a JS object.

**Comment by emmenko** [comment]
Given that I got two approvals and that all builds are green, I'm going ahead and merge this, so that we can unblock other PRs

**Pull request merged**